app/templates/services/_base_service_page.html

```diff
@@ -59,9 +59,6 @@
       {% import "toolkit/summary-table.html" as summary %}
       {% for section in sections %}
         {{ summary.heading(section.name, id=section.slug) }}
-        {% if section.editable %}
```

Comment:
e.g. it feels like the DOS-style per-lot questions we might want to show on a page together still:

## BEFORE
![screen shot 2017-02-10 at 11 39 35](https://cloud.githubusercontent.com/assets/6525554/22825523/b1c87abc-ef85-11e6-96a4-920f5ea5601d.png)

## NOW
![screen shot 2017-02-10 at 11 39 26](https://cloud.githubusercontent.com/assets/6525554/22825528/bc4763a4-ef85-11e6-88f9-17cce30d3137.png)